backends-velox/src/test/scala/io/glutenproject/execution/VeloxOrcDataTypeValidationSuite.scala

```diff
@@ -193,6 +194,14 @@ class VeloxOrcDataTypeValidationSuite extends VeloxWholeStageTransformerSuite {
   }
 
   test("Short type") {
+    // Validation: BatchScan with Filter
+    runQueryAndCompare(
+      "select type1.short, int from type1" +
+        " where type1.short = 1", false) { df: DataFrame =>
```

Comment:
Should we also test not equal cases?

Reply:
The test in this suite GlutenUnwrapCastInComparisonEndToEndSuite has all cases